pages/linux/rig.md

```diff
@@ -0,0 +1,24 @@
+# rig
+
+> Utility that will piece together a random first name, last name, street number and address, along with a geographically consistant (ie, they all match the same area) city, state, ZIP code, and area code.
```

Comment:
Perhaps this will read better?
```suggestion
> Utility to piece together a random first name, last name, street number and address, along with a geographically consistent (ie, they all match the same area) city, state, ZIP code, and area code.
```